Signal/src/util/OWSBackupAPI.swift

```diff
@@ -225,52 +225,55 @@ import CloudKit
     // MARK: - Delete
 
     @objc
-    public class func deleteRecordFromCloud(recordName: String,
+    public class func deleteRecordsFromCloud(recordNames: [String],
```

Comment:
Added bulk delete for cloud cleanup because the perf benefits are huge.

Reply:
We might want to do something similar for saves, downloads and "check if exists".